src/extensions/timeseries/__init__.py

```diff
@@ -0,0 +1 @@
+from .timeseries import *
```

Comment:
This is just to ensure that all the widgets get registered via decorator. It doesn't actually export anything in particular for consumption.